src/wh_utils.c

```diff
@@ -0,0 +1,32 @@
+#include "wolfhsm/wh_utils.h"
+
+static int isLittleEndian() {
+    unsigned int x = 1; /* 0x00000001 */
+    char *c = (char*)&x;
+    return (int)*c;
+}
```

Comment:
This should be a compile-time constant.  Let's add a backlog item to replace this with a macro based on some of the compiler defines and fallback to dynamic evaluation only if we can't determine it at compile time.